src/test/java/no/hvl/dat250/h2020/group5/controllers/AdminControllerTest.java

```diff
@@ -0,0 +1,154 @@
+package no.hvl.dat250.h2020.group5.controllers;
+
+import no.hvl.dat250.h2020.group5.dao.GuestRepository;
+import no.hvl.dat250.h2020.group5.dao.PollRepository;
+import no.hvl.dat250.h2020.group5.dao.UserRepository;
+import no.hvl.dat250.h2020.group5.entities.*;
+import no.hvl.dat250.h2020.group5.requests.UpdateUserRequest;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.ResponseEntity;
+import org.springframework.http.client.HttpComponentsClientHttpRequestFactory;
+
+import java.net.URL;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class AdminControllerTest {
+
+    @LocalServerPort private int port;
+
+    @Autowired TestRestTemplate template;
+
+    @Autowired AdminController adminController;
+
+    @Autowired PollRepository pollRepository;
+
+    @Autowired UserRepository userRepository;
+
+    @Autowired GuestRepository guestRepository;
+
+    private URL base;
+    private User user1;
+    private Poll poll1;
+
+    @BeforeEach
+    public void setUp() throws Exception {
```

Comment:
Hvis vi bruker @JpaTest så mener jeg dette skjer av seg selv, altså cleane repositoriet for hver test

Reply:
Ahh, ok, Skal sjekke det ut 👍

Reply:
Skal prøve å kommentere *før* jeg godkjenner neste gang haha, gikk litt fort her. 🥇

Reply:
Ja, jeg var litt kjapp på avtrekkeren selv også. På min pc fungerte det ikke med "DataJpaTest" til denne testklassen. Får en feilmelding.